model/src/main/proto/format_import_proto_library.bzl

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
The first line of the file doesn't look good empty, I guess. How about we can add some comment about what this file is all about. This helps other developers to understand about it.

Reply:
Okay! Fixed!